test/sjcl-ecdsa-signdeterministic-test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+/*eslint-disable new-cap*/
+
+var assert = require('assert');
+var sjcl = require('ripple-lib').sjcl;
+
+describe('Deterministic ECDSA (RFC 6979)', function() {
```

Comment:
You still haven't actually tested that it works. Please add the test cases provided in the RFC: https://tools.ietf.org/html/rfc6979#appendix-A.2.5